### PR TITLE
Refactoring, added wrapper around sys

### PR DESCRIPTION
**Pull request by europ**

### CODE REVIEW

File: system.py [hunk 1]
```diff
@@ -1,11 +1,35 @@
+"sys wrapper"
+
+
 import sys
 
 
+###############################################################################
```
Comment (tammar96) [inline system.py]: what is this?

File: const.py [hunk 2]
```diff
@@ -1,3 +1,6 @@
+"project constants"
```
Comment (tammar96) [inline const.py]: Why there must be an extra const file?

Comment (europ) [reply const.py]: It has been here since the beginning.

File: system.py (hunk 1 above)
Comment (europ) [reply system.py]: Separator, would you like to remove it?

File: const.py (hunk 2 above)
Comment (tammar96) [reply const.py]: Really? My bad.

File: system.py (hunk 1 above)
Comment (tammar96) [reply system.py]: yup

File: const.py (hunk 2 above)
Comment (europ) [reply const.py]: It was originally in main.py but after refactoring it was detached into a single file.

**Comment by xbolva00** [comment]
Seems ok

**Pull request merged**